bigquery2/bigquery_endpoint.bal

```diff
@@ -407,9 +408,9 @@ public remote function Client.runQuery(string projectId, @sensitive string query
 
 public remote function Client.getQueryResults(string projectId, string jobId, string nextPageToken = "")
                            returns @tainted QueryResults|error {
-    string getQueryResultsPath = string `{{PROJECTS_PATH}}/{{projectId}}/queries/{{jobId}}`;
+    string getQueryResultsPath = PROJECTS_PATH + "/" + projectId + "/queries/" + jobId;
     if (nextPageToken != "") {
-        getQueryResultsPath = string `{{getQueryResultsPath}}?pageToken={{nextPageToken}}`;
+        getQueryResultsPath = getQueryResultsPath + "?pageToken=" + nextPageToken;
```

Comment:
re: all the replacements for the string template literal, was there a reason for the change from string templates to concatenating with "+"?

Reply:
For this line 
https://github.com/wso2-ballerina/module-bigquery/pull/4/files#diff-2502bde90855403d573cec68559defb2L373

When I print the value of the path it goes as {{PROJECTS_PATH}}/{{projectId}}/queries/{{jobId}}
So from the back end i'm getting the error as below that's why I changed
Error: {wso2/bigquery2}BigQueryError {"message":"Error occurred while invoking the REST API"}

Reply:
The string template literal placeholder for interpolation was changed from `{{expr}}` to `${expr}` which is why you are observing the behaviour.

The following should work as expected:
```ballerina
string `${PROJECTS_PATH}/${projectId}/queries/${jobId}`
```

Reply:
Fixed